src/tcp-intercept.cxx

```diff
@@ -243,6 +244,14 @@ static void listening_socket_ready_for_read(EV_P_ ev_io *w, int revents) {
 	try {
 		new_con->s_client = s_listen->accept(&client_addr);
 
+		int val = 1;
+		//We do not want to buffer small packets, which could increase latency/gitter
```

Comment:
I think that would be the best option, but at the moment I do not know how  .... need to read about it.